### PR TITLE
Update to swift4.2 and sdk11

### PR DESCRIPTION
**Pull request by infiniteNIL**
This PR just updates the code to Swift 4.2 (eliminating some warnings) and updates the deployment target to iOS SDK 11.4

I also bumped the pod version to 7.0 in anticipation of more changes coming before we do an official release (which I'm not sure how to do)

### CODE REVIEW

File: Astro.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -20,10 +20,8 @@
...
 		835932AC62597B13CED887E4 /* ComparableSpec.swift in Sources */ = {isa = PBXBuildFile; fileRef = 83593E90B43887E2AE664086 /* ComparableSpec.swift */; };
 		835937E4D0DE85A9286BACC6 /* Comparable.swift in Sources */ = {isa = PBXBuildFile; fileRef = 83593EEFC7D26F39183B2AE1 /* Comparable.swift */; };
-		B4B3605221B725A800B7038F /* UIImage+AstroGadgets.swift in Sources */ = {isa = PBXBuildFile; fileRef = B4B3605121B725A800B7038F /* UIImage+AstroGadgets.swift */; };
-		B4B3605521B725B600B7038F /* Array+AstroGadgets.swift in Sources */ = {isa = PBXBuildFile; fileRef = B4B3605321B725B600B7038F /* Array+AstroGadgets.swift */; };
-		B4B3605621B725B600B7038F /* Bundle+AstroGadgets.swift in Sources */ = {isa = PBXBuildFile; fileRef = B4B3605421B725B600B7038F /* Bundle+AstroGadgets.swift */; };
 		B69726CE1BD9162F0047EEB9 /* LoggerSpec.swift in Sources */ = {isa = PBXBuildFile; fileRef = B69726CD1BD9162F0047EEB9 /* LoggerSpec.swift */; };
+		BD62EA481C929BF0004A02DF /* EnumCountable.swift in Sources */ = {isa = PBXBuildFile; fileRef = BD62EA471C929BF0004A02DF /* EnumCountable.swift */; };
```
Comment (interstateone) [inline Astro.xcodeproj/project.pbxproj]: I think the intent in #26 was to remove EnumCountable.swift (the corresponding spec file was removed) but it something might have been missed. Can you remove it from the project file here and delete EnumCountable.swift as part of this PR to clean that up?

File: Astro.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -20,10 +20,8 @@
 		638C62521BD454FD0004ECE1 /* Astro.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 638C62471BD454FD0004ECE1 /* Astro.framework */; };
 		835932AC62597B13CED887E4 /* ComparableSpec.swift in Sources */ = {isa = PBXBuildFile; fileRef = 83593E90B43887E2AE664086 /* ComparableSpec.swift */; };
 		835937E4D0DE85A9286BACC6 /* Comparable.swift in Sources */ = {isa = PBXBuildFile; fileRef = 83593EEFC7D26F39183B2AE1 /* Comparable.swift */; };
-		B4B3605221B725A800B7038F /* UIImage+AstroGadgets.swift in Sources */ = {isa = PBXBuildFile; fileRef = B4B3605121B725A800B7038F /* UIImage+AstroGadgets.swift */; };
-		B4B3605521B725B600B7038F /* Array+AstroGadgets.swift in Sources */ = {isa = PBXBuildFile; fileRef = B4B3605321B725B600B7038F /* Array+AstroGadgets.swift */; };
-		B4B3605621B725B600B7038F /* Bundle+AstroGadgets.swift in Sources */ = {isa = PBXBuildFile; fileRef = B4B3605421B725B600B7038F /* Bundle+AstroGadgets.swift */; };
```
Comment (interstateone) [inline Astro.xcodeproj/project.pbxproj]: I think these files should be part of the project now that #26 was merged.

**Comment by infiniteNIL** [comment]
Shouldn't I be seeing some conflicts in regards to your comments?

**Comment by interstateone** [comment]
I don't think they're conflicts. It looks like your changes on this branch revert some of the changes that were merged with #26. E.g. a [file that was added](https://github.com/RobotsAndPencils/Astro/pull/26/files#diff-1f8baf77201d8b37fe5432382270155bR23) to the project file in #26 looks like it's [removed](https://github.com/RobotsAndPencils/Astro/pull/27/files/40d51fe393d1a32ee0c11af1987db1ae21c37061#diff-1f8baf77201d8b37fe5432382270155bL23) as part of this branch. This happened in the last commit, but I'm not sure how "Remove invalid test" relates to the changes to the project file. Can you clarify that?

**Pull request closed without merging**